### ISSUE TITLE
Imported wallets are not supported by Label sync plugin

### ISSUE THREAD

**Issue by ysangkok** [spesmilo/electrum#6703]
Using master.

```
  File "electrum/plugins/labels/labels.py", line 105, in push_thread
    raise Exception('Wallet {} not loaded'.format(wallet))
```

This is even though all transactions are showing fine. This is an imported wallet.

**Comment by SomberNight** [comment]
> This is an imported wallet.

The root cause is that imported wallets are not supported by the Labels plugin.

https://github.com/spesmilo/electrum/blob/193c29af87406646c7db54b0095a2c88ad6d8553/electrum/plugins/labels/labels.py#L177-L188

`wallet_id` is calculated from the xpub (or concat of xpubs for multisig), and it is sent to the Labels server as an identifier for your labels

**Comment by SomberNight** [comment]
I've fixed your trace in https://github.com/spesmilo/electrum/commit/c616c3bfad6954e39192e24d9e1695822209bcb6.

Not sure if the purpose of your report was just the specific trace (that there is a UI flow to trigger the exception) or if it was the more general issue of there not being Label Sync support for Imported wallets.
If the former, it's fixed now.
If the latter, I don't see how it would be possible. Any wallet ID that we derive from imported addresses would presumably change if the user imported more addresses. If we generated an ID independent of addresses and stored it in the wallet file, there is no point of syncing labels as, as long as the user has the file, the user has the labels too (they are in the file).
